src/main/java/com/jme3/math/Quaternion.java

```diff
@@ -1488,14 +1493,21 @@ public Quaternion inverseLocal() {
 	}
 
 	/**
-	 * <code>negate</code> inverts the values of the quaternion.
-	 *
+	 * <code>negate</code> inverts the values of the quaternion. and returns it
+	 */
+	public Quaternion negateLocal() {
+		x = -x;
+		y = -y;
+		z = -z;
+		w = -w;
+		return this;
+	}
+
+	/**
+	 * <code>negate</code> returns the negated quaternion.
```

Comment:
Maybe this could be a bit clearer?
```suggestion
	 * <code>negate</code> returns a negated copy of the quaternion.
```